src/.vscode/extensions.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Do we want this file given that we have `.vscode/extensions.json` at the root of the repo too?

Reply:
Good catch!  Don't know how that got there.  Removed.